core/field_dropdown.js

```diff
@@ -68,15 +68,23 @@ Blockly.FieldDropdown = function(menuGenerator, opt_validator) {
   this.menuGenerator_ = menuGenerator;
 
   /**
-   * The currently selected index. A value of -1 indicates no option
-   * has been selected.
+   * A cache of the most recently generated options.
+   * @type {Array.<!Array.<string>>}
```

Comment:
I think `.` in array declarations is deprecated.

Reply:
It is, but we're keeping it in Blockly for consistency.